anomalib/models/fastflow/config.yaml

```diff
@@ -0,0 +1,102 @@
+dataset:
+  name: mvtec #options: [mvtec, btech, folder]
+  format: mvtec
+  path: ./datasets/MVTec
+  task: segmentation
+  category: wood
+  image_size: 256 # options: [256, 256, 448, 384] - for each supported backbone
+  train_batch_size: 32
+  test_batch_size: 32
+  num_workers: 0
```

Comment:
`num_workers` is usually set to a higher default value, for the other models it's 36.

Reply:
thanks for spotting this